src/login.js

```diff
@@ -0,0 +1,87 @@
+import { getEmailErrorMsg, getPasswordErrorMsg, getNicknameErrorMsg } from './validationErrorMessage.js';
+
+// Elements
+const emailInput = document.querySelector('#email');
+const emailError = document.querySelector('#email + .input-validation-error');
```

Comment:
P1:
지금으로는 크게 헷갈리지 않지만 기존의 변수명인 `emailError`의 경우 에러 메시지인지 에러 메시지가 나올 태그인지 알기어렵습니다.
아래처럼 좀 더 명확하게 명명해주시면 좋을 것 같습니다.
(제가 네이밍에는 크게 자신이 없어서 참고만해주세요~)

```suggestion
const emailErrorContainer = document.querySelector('#email + .input-validation-error');
```